### PR TITLE
Rust 1.72 upgrade

### PR DESCRIPTION
**Pull request by Anton-4**

### CODE REVIEW

**Comment by Anton-4** [comment]
The failing tests are hitting #5924

**Comment by bhansconnect** [comment]
I think we should just disable the 2 tests for now on linux-x86-64 and merge this.

File: crates/glue/src/glue.rs [hunk 1]
```diff
@@ -2971,20 +2971,7 @@ impl Clone for U4 {
...
-                discriminant_U4::Some => Self {
-                    Some: self.Some.clone(),
-                },
-            }
-        };
-
-        answer.set_discriminant(self.discriminant());
-
-        answer
+        *self
```
Comment (bhansconnect) [inline crates/glue/src/glue.rs:2974]: Are we confident all of these changes are correct?

**Review by bhansconnect — APPROVED** [review]

File: crates/glue/src/glue.rs (hunk 1 above)
Comment (Anton-4) [reply crates/glue/src/glue.rs:2974]: I checked a similar change with Richard and he said it was ok.

**Pull request merged**